scalikejdbc-syntax-support-macro/src/test/scala/foo/AutoSpec.scala

```diff
@@ -14,9 +14,10 @@ class AutoSpec extends FlatSpec with Matchers with DBSettings {
     def apply(r: ResultName[Organization])(rs: WrappedResultSet): Organization = autoConstruct(rs, r)
   }
 
-  case class Person(id: Long, name: String, organizationId: Option[Long], organization: Option[Organization] = None, gorupId: Long = 0)
+  case class Person(id: Long, name: String, organizationId: Option[Long], organization: Option[Organization] = None, groupId: Long = 0)
   object Person extends SQLSyntaxSupport[Person] {
     def apply(s: SyntaxProvider[Person])(rs: WrappedResultSet): Person = autoConstruct(rs, s, "organization")
+    override lazy val columns = useField[Person]("organization")
```

Comment:
>I agree that the FQCN seemes strange, however I think this API is a kind of DSL.
So autoConstruct breaks standard naming convention(starts upper case).

I meant it's a bit strange that `entityFields` returns column names (and it's a public API).

Thanks. After you rename that, it's possible to merge this PR. 

I'm planning to release scalikejdbc 2.2.7 with a fix on scalikejdbc-config to support both of user and userame due to Play 2.4's config renaming soon. I'd love to include this great feature at the time.